merlin/celery.py

```diff
@@ -226,8 +226,9 @@ def handle_worker_startup(sender: str = None, **kwargs):
                 # Sender name is of the form celery@worker_name.%hostname
                 worker_name, host = sender.split("@")[1].split(".%")
                 merlin_db = MerlinDatabase()
-                logical_worker = merlin_db.get_logical_worker(worker_name=worker_name, queues=options.get("queues"))
-                physical_worker = merlin_db.create_physical_worker(
+                logical_worker = merlin_db.create("logical_worker", name=worker_name, queues=options.get("queues"))
```

Comment:
just curious why are you creating now and before it seemed to have been there

Reply:
I think this might've been a typo, good catch.